hello_world.py

```diff
@@ -9,7 +11,7 @@
 
 
 envs = MicroRTSGridModeVecEnv(
-    num_selfplay_envs=0,
+    num_selfplay_envs=2,
```

Comment:
Any reason for this change?

Reply:
No, I just wanted to see what happens if it is not zero.